crates/bonsaidb-core/src/networking.rs

```diff
@@ -84,6 +84,12 @@ pub enum ServerRequest {
         /// The unique username of the user to create.
         username: String,
     },
+    /// Deletes a user.
+    #[cfg_attr(feature = "actionable-traits", actionable(protection = "simple"))]
+    DeleteUser {
+        /// The unique primary key of the user to be deleted.
+        primary_key: NamedReference<'static, u64>,
```

Comment:
To match the other networking requests, let's rename this to `user`.

Reply:
Done.